get_asteroidcount.py

```diff
@@ -0,0 +1,57 @@
+from astroquery.mpc import MPC
+
+class AsteroidWidget():
+
+    def __init(self,
+             longitude: Optional[float] = None,
```

Comment:
Indentation should be the same for all constructor arguments, otherwise a [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation)-based linter will complain about _"continuation line under-indented for visual indent"_.